hott-letter-exercises.tex

```diff
@@ -0,0 +1,31 @@
+% -*- mode: LaTeX -*-
+% This is a main LaTeX file for generating the HoTT book
+% for printing on letter paper. The links are black in this
+% version.
```

Comment:
Do you want to add something to the comment mentioning the one-per-page aspect of this file?

Reply:
>  Is the idea to use these as group-work exercises in a class or something?
I think it would be helpful for that.
My motivation was for myself, working through the exercises, and I hoped it might be helpful to others.